db/migrate/20200219224408_add_role_to_users.rb

```diff
@@ -1,5 +1,5 @@
 class AddRoleToUsers < ActiveRecord::Migration[5.1]
   def change
-    add_column :users, :role, :integer, default: 0
+    add_column :users, :role, :integer, default: 1
```

Comment:
Default user should not be a visitor. If a user is created, it should be the default user which is role 1 for us.